lib/vsac-code-systems.json

```diff
@@ -1,45 +1,45 @@
 ﻿{
-  "2.16.840.1.113883.12.112":   { "name": "DischargeDisposition" },
+  "2.16.840.1.113883.12.112":   { "name": "DischargeDisposition" ,  "uri": "http://hl7.org/fhir/discharge-disposition" },

```

Comment:
I don't think the code system `http://hl7.org/fhir/discharge-disposition` is the same as the one for OID `2.16.840.1.113883.12.112`.  The [doc](http://hl7.org/fhir/STU3/codesystem-encounter-discharge-disposition.html) for the former lists a different OID than the latter -- and the codes appear to be different than the ones listed [here](https://phinvads.cdc.gov/vads/ViewCodeSystem.action?id=2.16.840.1.113883.12.112).

So... I think we should remove the uri from this one.

Reply:
I see now that VSAC thinks this relationship (between URI and OID) is valid.  Well... we're in a pickle now.

Reply:
I think our only choice is to not include the offending uri.  I'll do that now.